### PR TITLE
COPS-953: fix the way we fetch proxysql password in hn_nodes_list.py.

### PR DESCRIPTION
**Pull request by Ronan-Gorain**
Handle storage nodes in stop/start-node.yaml

JIRA issue: https://jira.jahia.org/browse/COPS-953

Short description:

**Don't forget** to update the README and any related documentation if needed: https://confluence.jahia.com/x/bYATAg

### CODE REVIEW

File: assets/common/hn_nodes_list.py [hunk 1]
```diff
@@ -92,7 +92,7 @@ def check_if_db_master_node(envname, node_index, jelastic_user_session):
         Return True if the db node is the current cluster "master", False otherwise
     """
     url = jelastic_user_session.hostname + "/1.0/environment/control/rest/execcmdbygroup"
-    command = "mysql -NB -u admin -padmin -P 6032 -h 127.0.0.1 " \
+    command = "mysql -NB -u admin -p$(grep password /etc/datadog-agent/conf.d/proxysql.d/conf.yaml| awk '{print $2}') -P 6032 -h 127.0.0.1 " \
```
Comment (laurentfufu) [inline assets/common/hn_nodes_list.py:95]:
```suggestion
    command = "mysql -NB -u admin -p$(awk '$1=="password:" '{print $2}' /etc/datadog-agent/conf.d/proxysql.d/conf.yaml) -P 6032 -h 127.0.0.1 " \
```

Comment (Ronan-Gorain) [reply assets/common/hn_nodes_list.py]: updated

File: assets/common/hn_nodes_list.py [hunk 2]
```diff
@@ -92,7 +92,7 @@ def check_if_db_master_node(envname, node_index, jelastic_user_session):
         Return True if the db node is the current cluster "master", False otherwise
     """
     url = jelastic_user_session.hostname + "/1.0/environment/control/rest/execcmdbygroup"
-    command = "mysql -NB -u admin -padmin -P 6032 -h 127.0.0.1 " \
+    command = "mysql -NB -u admin -p$(awk '$1=="password:" '{print $2}' /etc/datadog-agent/conf.d/proxysql.d/conf.yaml) -P 6032 -h 127.0.0.1 " \
```
Comment (laurentfufu) [inline assets/common/hn_nodes_list.py:95]:
```suggestion
    command = "mysql -NB -u admin -p$(awk '$1=="password:" {print $2}' /etc/datadog-agent/conf.d/proxysql.d/conf.yaml) -P 6032 -h 127.0.0.1 " \
```
:grin:

**Review by laurentfufu — CHANGES_REQUESTED** [review]

File: assets/common/hn_nodes_list.py (hunk 2 above)
Comment (laurentfufu) [reply assets/common/hn_nodes_list.py:95]: never trust me :trollface:

Comment (Ronan-Gorain) [reply assets/common/hn_nodes_list.py]: Fixed :)

**Pull request merged**